domains/certificates/RegistryConnector/Worker/RetryOptions.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace RegistryConnector.Worker;
+
+public class RetryOptions
+{
+    public const string Retry = nameof(Retry);
+
+    [Required]
+    public int DefaultFirstLevelRetryCount { get; set; }
```

Comment:
What is firstlevel and secondlevel?

Reply:
First level tries right away, second level tries again after some time: https://masstransit.io/documentation/concepts/exceptions